docs/namespace.md

```diff
@@ -0,0 +1,65 @@
+---
+id: namespace
+title: Customizing Namespace
+sidebar_label: Customizing Namespace
+---
+
+OpenEBS is usually deployed in the *default* namespace. You can deploy OpenEBS in another namespace  other than the *default* namespace.  To customize namespace to *openebs* instead of *default*,  you must change all the namespaces in the *openebs-operator.yaml* file to *openebs*. 
+
+1. Create a namespace as in the following example i.e., modify *name:* to *openebs*.
+
+   ```
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: openebs
+   ```
+
+2. Change all occurrences of *namespace: default* to *namespace: openebs* in the *openebs-operator.yaml* file.
```

Comment:
Looks redundant, can it be removed here or from the earlier point?

Reply:
Done